akka-cluster/src/main/scala/akka/cluster/MembershipState.scala

```diff
@@ -22,10 +22,14 @@ import akka.util.ccompat._
 @ccompatUsedUntil213
 @InternalApi private[akka] object MembershipState {
   import MemberStatus._
-  private val leaderMemberStatus = Set[MemberStatus](Up, Leaving)
+  private val leaderMemberStatus = Set[MemberStatus](Up, Leaving, PreparingForShutdown, ReadyForShutdown)
+  // FIXME should shutting down be added to these?
   private val convergenceMemberStatus = Set[MemberStatus](Up, Leaving)
```

Comment:
Used in `def convergence`. As far as I can see they should be included here, since they are similar to Up (they were Up).

Reply:
Agreed